packages/wrangler/e2e/unenv-preset/preset.test.ts

```diff
@@ -54,6 +55,19 @@ const testConfigs: TestConfig[] = [
 				enable_nodejs_http_modules: true,
 			},
 		},
+		{
+			name: "http server enabled by flag",
+			compatibilityDate: "2025-07-26",
+			compatibilityFlags: [
+				"enable_nodejs_http_modules",
+				"enable_nodejs_http_server_modules",
+				"experimental",
```

Comment:
AFAIK you cannot actually send the `experimental` flag to production workers. It is not allowed in edgeworker. So I assume that this will need to change once we get the server_modules flags into production.

Reply:
TIL, then we need to skip experimental remote test.

Reply:
FWIW the `experimental` flag is always passed to Miniflare's workerd instances because it is needed to enable the loopback server and other things. So it might just be fine to remove it here altogether.

Reply:
> FWIW the `experimental` flag is always passed to Miniflare's workerd instances because it is needed to enable the loopback server and other things. So it might just be fine to remove it here altogether.

If we remove `experimental` then we don't have a way to skip the test on remote. So I would prefer to keep it there.

That way users can test experimental feature locally - they will get an error when tying to deploy because of the `experimental` flag.

WDYT?

Reply:
I don't mind either way, since before we land this PR the experimental flag will be unnecessary.

Reply:
We will skip experimental tests on remote.